src/main/webapp/app/exercises/modeling/shared/modeling-editor.component.scss

```diff
@@ -49,7 +49,24 @@ $draggable-width: 15px;
 
 .readonly-diagram {
     height: 100%;
-    overflow: hidden;
+    overflow-x: auto;
+    overflow-y: auto;
+}
+
+.scroll-container {
+    scrollbar-width: thin; /* Firefox */
+    &::-webkit-scrollbar {
+        width: 6px; /* WebKit */
```

Comment:
Ideally we avoid px if possible otherwise i dont have a too biased opinion on that